src/ripple/app/ledger/Ledger.cpp

```diff
@@ -273,14 +260,13 @@ Ledger::Ledger (open_ledger_t, Ledger const& prevLedger,
     , fees_(prevLedger.fees_)
     , rules_(prevLedger.rules_)
 {
-    info_.open = true;
     info_.seq = prevLedger.info_.seq + 1;
     info_.parentCloseTime =
         prevLedger.info_.closeTime;
     info_.hash = prevLedger.info().hash + uint256(1);
     info_.drops = prevLedger.info().drops;
     info_.closeTimeResolution = prevLedger.info_.closeTimeResolution;
-    info_.parentHash = prevLedger.getHash ();
+    info_.parentHash = prevLedger.info().hash;
```

Comment:
There is an entire set of commits that replace `ledger.getHash()` with `ledger.info().hash`. All else being equal I prefer a style that stays one level deep. What motivated this change?